tests/test_py3_codequality.py

```diff
@@ -5,17 +5,3 @@
 Code Style Tests.
 """
 import subprocess
-
-
-def test_check_python_style():
-    files = ('./', )
-    cmd = ('yapf', '--diff', '--recursive', '--exclude', './.git/*')
-    passed = len(subprocess.check_output(cmd + files)) == 0
-    assert passed, 'The python code does not adhear to the project style.'
-
-
-def test_check_python_flake8():
-    files = ('.', )
-    cmd = ('flake8', )
-    passed = subprocess.call(cmd + files) == 0
-    assert passed, 'Flake8 did not run cleanly.'
```

Comment:
more overlap with #99 - make sure this change to remove style tests doesn't land.

Reply:
This change will not land in master, but will be updated with what's in #99. The reason this was removed was that the updated versions of `yapf` and `flake8` conflict with each other in numerous ways and we've decided to switch to black to match with lando-api.